mne_bids/write.py

```diff
@@ -1352,3 +1353,163 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
     nib.save(t1w, t1w_basename)
 
     return anat_dir
+
+
+def mark_bad_channels(ch_names, descriptions=None, *, bids_basename,
+                      bids_root, kind=None, overwrite=False, verbose=True):
+    """Update which channels are marked as "bad" in an existing BIDS dataset.
+
+    Parameters
+    ----------
+    ch_names : str | list of str
+        The names of the channel(s) to mark as bad. Pass an empty list in
+        combination with ``overwrite=True`` to mark all channels as good.
+    descriptions : None | str | list of str
+        Descriptions of the reasons that lead to the exclusion of the
+        channel(s). If a list, it must match the length of ``ch_names``.
+        If ``None``, no descriptions are added.
+    bids_basename : BIDSPath | str
+        The base filename of the BIDS compatible files. Typically, this can be
+        generated using :func:`mne_bids.make_bids_basename`.
+    bids_root : str | pathlib.Path
+        Path to root of the BIDS folder.
+    kind : str | None
+        The kind of recording to update. If ``None`` and only one kind (e.g.,
+        only EEG or only MEG data) is present in the dataset, it will be
+        selected automatically.
+    overwrite : bool
+        If ``False``, only update the information of the channels passed via
+        ``ch_names``, and leave the rest untouched. If ``True``, update the
+        information of **all** channels: mark the channels passed via
+        ``ch_names`` as bad, and all remaining channels as good, also
+        discarding their descriptions.
+    verbose : bool
+        The verbosity level.
+
+    Raises
+    ------
+    ValueError
+        If the numbers of channels and descriptions do not match.
+
```

Comment:
can we not have blank lines betwen the items in this "raises" list?

Reply:
Actually … all these exceptions are kind of "obvious", so I've now decided to remove the entire `Raises` section.